kibana-notebooks/.cypress/integration/ui.spec.js

```diff
@@ -274,7 +274,7 @@ describe('Testing paragraphs', () => {
     cy.wait(delay);
     cy.get('.euiContextMenuItem__text').contains('Rename notebook').click();
     cy.wait(delay);
-    cy.get('input.euiFieldText').type(' (rename)');
+    cy.get('input.euiFieldText[data-autofocus="true"]').type(' (rename)');
```

Comment:
Just curious, what does the `[data-autofocus="true"]` flag change in this test?

Reply:
There might be other `input.euiFieldText` in embedded visualizations on the page, this makes sure cypress selects the input box inside the rename toast. Using custom class/id selector is better, but none of the `euiFieldText` in sample visualizations has autofocus, so this works as well